README.md

```diff
@@ -218,4 +218,4 @@ Pls see the [License](./LICENSE.md) file.
 
 # Contact Us
 
-`DPVS` is developing by [iQiYi](www.iqiyi.com) [QLB](mailto:qlb-devel@dev.qiyi.com) team since Arpil 2016 and now open-sourced. It's already widely used in iQiYi IDC for L4 load balancer and SNAT clusters, and we plan to replace all our LVS clusters with DPVS. We are very happy that **more people can get involved** in this project. Welcome to try, report issues and submit pull requests. And pls feel free to contact us through **Github** or [Email](mailto:qlb-devel@dev.qiyi.com).
+`DPVS` is developing by [iQiYi](https://www.iqiyi.com) [QLB](mailto:qlb-devel@dev.qiyi.com) team since Arpil 2016 and now open-sourced. It's already widely used in iQiYi IDC for L4 load balancer and SNAT clusters, and we plan to replace all our LVS clusters with DPVS. We are very happy that **more people can get involved** in this project. Welcome to try, report issues and submit pull requests. And pls feel free to contact us through **Github** or [Email](mailto:qlb-devel@dev.qiyi.com).
```

Comment:
http://www.iqiyi.com is better for me since www.iqiyi.com is not fully secured by https so far, your browser will notice you if you access iqiyi through https protocol, like below

![screen shot 2017-10-10 at 22 39 41](https://user-images.githubusercontent.com/689798/31392669-7d7e52bc-ad9f-11e7-98dc-eff0abb0bad2.png)